src/components/Icon/Icon.stories.tsx

```diff
@@ -28,12 +28,16 @@ export default {
       description:
         "Accessible icon name used by screen readers and other assistive technologies. Provide when icon is not merely decorative",
     },
+    renderWhiteIcon: {
+      description: "Renders a white icon instead of the default black icon",
+    },
   },
   args: {
     name: "accessibility_new",
     iconSize: 3,
     ariaHidden: true,
     role: "img",
+    renderWhiteIcon: false,
```

Comment:
This got me thinking of a feature `renderInverse` or something like that. Essentially render the SVG in the inverse color of the background. There might not be as much application for that as I think there is, but the idea seems cool to me right now. It's also probably a way bigger lift than is worth.